File: Iniciante/URI 1015.go
package main
 
import (
    "fmt"
    "math"
)

func main() {
    var x1, y1, x2, y2, distancia float64

    fmt.Scanf("%f", &x1)
    fmt.Scanf("%f", &y1)
    fmt.Scanf("%f", &x2)
    fmt.Scanf("%f", &y2)
    distancia = math.Sqrt(math.Pow((x2 - x1), 2.0) + math.Pow((y2 - y1), 2.0))
    fmt.Printf("%.4f\n", distancia)
}

File: Iniciante/URI 1015.scala
import java.util.Scanner

object Main {

    def main(args: Array[String]) {
        val leitor = new Scanner(System.in)
        val x1 = leitor.nextDouble()
        val y1 = leitor.nextDouble()
        val x2 = leitor.nextDouble()
        val y2 = leitor.nextDouble()
        val distancia = Math.sqrt(Math.pow((x2 - x1), 2.0) + Math.pow((y2 - y1), 2.0))
        printf("%.4f\n", distancia)
    }
	
}